hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -676,12 +682,15 @@ cdef class ArbitrageStrategy(StrategyBase):
                                                  str buy_market_quote_currency,
                                                  str sell_market_quote_currency):
         """
+        Iterates through sell and buy order books and returns an array of matched profitable sell and buy order
+        pairs.
         :param min_profitability: 
         :param buy_order_book: 
         :param sell_order_book: 
         :param buy_market_quote_currency: 
         :param sell_market_quote_currency: 
-        :return: bid_price, ask_price, amount
+        :return: ordered array of discrete, profitable combinations of sell and buy orders from the bid and ask
```

Comment:
dont think we should remove original return, maybe add this line as a separate comment

Reply:
I agree. The second line is more confusing than the first